crates/pathfinder/src/sequencer.rs

```diff
@@ -3,15 +3,19 @@ pub mod error;
 pub mod reply;
 pub mod request;
 
-use self::error::StarknetError;
-use crate::{
-    core::{ContractAddress, StarknetTransactionHash, StorageAddress, StorageValue},
-    ethereum::Chain,
-    rpc::types::{BlockHashOrTag, BlockNumberOrTag, Tag},
-    sequencer::error::SequencerError,
-};
+use std::borrow::Cow;
```

Comment:
Just wondering - why split imports into several lines with surplus `use`s? `rustfmt` will organize all of them in alphabetical order (except for `self` and `super`, if I recall correctly, being the topmost).

Reply:
That was actually `rustfmt` doing that.. I think its because I set it to use some nightly features, can revert if needed?

Reply:
Hehe, I think this is the more natural style; I've found these huge multiline things really weird and hard to read. It's not like we didn't have the disk space.

Shame if there's now a bit of breakage with formatting once more. (If you don't do formatting via rust-analyzer you can keep using the stable rustfmt while using nightly rust-analyzer.)

Ofc no matter the tool, you aren't forced to include the changes a tool makes, and removing extra changes is always better.

Yeah seems that most of the changes are extra changes.

Reply:
Let's agree on some uniform formatting config, whichever this is :smile:

Reply:
I'll revert the formatting, and then I think we can have / create a `.rustfmt.toml` or whatever :)

I agree that I like the single lines better. Much easier to parse.

Reply:
Removed the formatting changes.